pkg/sql/distsqlrun/server.go

```diff
@@ -32,6 +32,38 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/util/tracing"
 )
 
+// MajorVersion and MinorVersion identify the distsqlrun protocol version.
+//
+// The planner populates major and minor versions in SetupFlorRequest.
+// A server only accepts requests with:
+//  - the same MajorVersion, and
+//  - a MinorVersion in the range MinAcceptedMinorVersion to MinorVersion.
+//
+// This can be used to provide a "window" of compatibility when new features are
+// added.
+//
+// Example:
+//  - we start with MinorVersion=1; distsqlrun servers with minor version 1
+//    only accept requests with minor version 1.
```

Comment:
nit: I don't really mind this but it might be good to keep using `MinorVersion` instead of switching between `minor version` and `MinorVersion.